.devcontainer/.env.example

```
@@ -0,0 +1,4 @@
+# Claude Code (Vertex AI auth)
+CLAUDE_CODE_USE_VERTEX=1
+ANTHROPIC_VERTEX_PROJECT_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ANTHROPIC_VERTEX_PROJECT_ID key should go before the CLAUDE_CODE_USE_VERTEX key

(UnorderedKey)